src/cow-react/modules/limitOrders/containers/QuoteResolver/hooks/useQuoteRequestParams.ts

```diff
@@ -0,0 +1,30 @@
+import { useLimitOrdersTradeState } from 'cow-react/modules/limitOrders/hooks/useLimitOrdersTradeState'
+import { useWeb3React } from '@web3-react/core'
+import { LegacyFeeQuoteParams as FeeQuoteParams } from 'api/gnosisProtocol/legacy/types'
+import { OrderKind } from '@cowprotocol/contracts'
+import { Token } from '@uniswap/sdk-core'
+import { parseUnits } from '@ethersproject/units'
+import { useMemo } from 'react'
+
+export function useQuoteRequestParams(): FeeQuoteParams | null {
```

Comment:
https://github.com/cowprotocol/cowswap/pull/1124#discussion_r983752396

> Limit orders won't need the fee. Just the price, but not the fee because it will be zero (once its implemented).
> 
> We might want to get the fee if they are rejecting our orders... (just temporary, once implemented they will start adding it)

@anxolin Currently, if we specify a low fee amount, then API response with the error:
```
description: "Order does not include sufficient fee"
errorType: "InsufficientFee"
```

Reply:
Yep, good to add for now. Also, I believe the backend will add the possibility to create limit orders (at least in goerli and for some whitelisted addresses)